flang/test/Lower/OpenMP/shared-loop.f90

```diff
@@ -0,0 +1,33 @@
+! RUN: bbc -emit-hlfir -fopenmp %s -o - | FileCheck %s
+! RUN: %flang_fc1 -emit-hlfir -fopenmp %s -o - | FileCheck %s
+
+! CHECK:  func.func @_QQmain() attributes
+! CHECK:    %[[ALLOC_I:.*]] = fir.alloca i32 {bindc_name = "i", uniq_name = "_QFEi"}
+! CHECK:    %[[DECL_I:.*]]:2 = hlfir.declare %[[ALLOC_I]] {uniq_name = "_QFEi"} :
+! CHECK:    omp.parallel {
+! CHECK:      omp.sections {
+! CHECK:        omp.section {
+! CHECK:          %[[RES:.*]]:2 = fir.do_loop {{.*}} iter_args(%[[ARG:.*]] =
```

Comment:
Please could you expend the variables here so we can see what exactly is being used as the loop iteration variable (I know we have the correct store at the end, but I think making it more explicit makes it clearer what the test is checking).

Reply:
Updated with a few more lines.

Reply:
Sorry I meant the arguments to fir.do_loop so we can see the variables we are storing to are actually the loop induction variable